applications/FluidDynamicsApplication/tests/embedded_couette_test.py

```diff
@@ -359,13 +357,18 @@ def checkResults(self):
 if __name__ == '__main__':
     test = EmbeddedCouetteTest()
     test.setUp()
-    test.distance = 0.25
-    test.slip_flag = True
-    test.print_output = True
-    test.print_reference_values = True
-    test.work_folder = "EmbeddedCouette2DTest"
-    test.reference_file = "reference_couette_embedded_slip_2D"
-    test.settings = "EmbeddedCouette2DTestParameters.json"
+    # test.testEmbeddedCouette2D()
+    # test.testEmbeddedCouette3D()
+    # test.testEmbeddedSlipCouette2D()
+    # test.testEmbeddedSlipCouette3D()
+    # test.testEmbeddedAusasCouette2D()
+    # test.testEmbeddedAusasCouette3D()
+    test.testEmbeddedDevelopmentCouette2D()
```

Comment:
why just one of the tests?

Reply:
I use this when I need to run only one of the tests. We can change it to run all the `EmbeddedCouetteTests` but I find this specially useful when debugging.